### ISSUE TITLE
[BUG] BadSignature('Signature "..." does not match') on document import

### ISSUE THREAD

**Issue by amo13** [jonaswinkler/paperless-ng#1110]
**Describe the bug**
When trying to import PDFs into a fresh installation of Paperless-ng on Archlinux, the procedure gets stuck after uploading and produces below lines in the scheduler log.

**To Reproduce**
1. Clean installation of paperless-ng
2. Launch the webserver, consumer and scheduler
3. Drop a PDF onto the web interface

**Expected behavior**
Process the imported PDF

**Webserver logs**
```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 351, in pusher
    task = SignedPackage.loads(task[1])
  File "/usr/lib/python3.9/site-packages/django_q/signing.py", line 25, in loads
    return signing.loads(
  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 40, in loads
    base64d = force_bytes(TimestampSigner(key, salt=salt).unsign(s, max_age=max_age))
  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 75, in unsign
    result = super(TimestampSigner, self).unsign(value)
  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 60, in unsign
    raise BadSignature('Signature "%s" does not match' % sig)
django.core.signing.BadSignature: Signature "FA9yQBJYTC2IPm6pryl5MV0kF4R9t1-SlWVX8W_i5BI" does not match
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.9/logging/__init__.py", line 1083, in emit
    msg = self.format(record)
  File "/usr/lib/python3.9/logging/__init__.py", line 927, in format
    return fmt.format(record)
  File "/usr/lib/python3.9/logging/__init__.py", line 663, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.9/logging/__init__.py", line 367, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/usr/bin/paperless-manage", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/lib/python3.9/site-packages/django_q/management/commands/qcluster.py", line 22, in handle
    q.start()
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 75, in start
    self.sentinel.start()
  File "/usr/lib/python3.9/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/usr/lib/python3.9/multiprocessing/context.py", line 277, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 71, in _launch
    code = process_obj._bootstrap(parent_sentinel=child_r)
  File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 165, in __init__
    self.start()
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 169, in start
    self.spawn_cluster()
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 243, in spawn_cluster
    self.pusher = self.spawn_pusher()
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 198, in spawn_pusher
    return self.spawn_process(pusher, self.task_queue, self.event_out, self.broker)
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 194, in spawn_process
    p.start()
  File "/usr/lib/python3.9/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/usr/lib/python3.9/multiprocessing/context.py", line 277, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 71, in _launch
    code = process_obj._bootstrap(parent_sentinel=child_r)
  File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 353, in pusher
    logger.error(e, traceback.format_exc())
Message: BadSignature('Signature "FA9yQBJYTC2IPm6pryl5MV0kF4R9t1-SlWVX8W_i5BI" does not match')
Arguments: ('Traceback (most recent call last):\n  File "/usr/lib/python3.9/site-packages/django_q/cluster.py", line 351, in pusher\n    task = SignedPackage.loads(task[1])\n  File "/usr/lib/python3.9/site-packages/django_q/signing.py", line 25, in loads\n    return signing.loads(\n  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 40, in loads\n    base64d = force_bytes(TimestampSigner(key, salt=salt).unsign(s, max_age=max_age))\n  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 75, in unsign\n    result = super(TimestampSigner, self).unsign(value)\n  File "/usr/lib/python3.9/site-packages/django_q/core_signing.py", line 60, in unsign\n    raise BadSignature(\'Signature "%s" does not match\' % sig)\ndjango.core.signing.BadSignature: Signature "FA9yQBJYTC2IPm6pryl5MV0kF4R9t1-SlWVX8W_i5BI" does not match\n',)
```

**Relevant information**
 - Host OS of the machine running paperless: Archlinux
 - Browser: Firefox
 - Version 1.4.4
 - Installation method: bare metal
 - Any configuration changes you made in `paperless.conf`:
 ```
PAPERLESS_REDIS=redis://localhost:6379
PAPERLESS_CONSUMPTION_DIR=/var/lib/paperless/consume
PAPERLESS_DATA_DIR=/var/lib/paperless/data
PAPERLESS_MEDIA_ROOT=/var/lib/paperless/media
PAPERLESS_SECRET_KEY=46c8d71de1c077e702bdkuzg42guk24
```

**Comment by amo13** [comment]
Just found out that the problem occurs only since 1.4.4. I just installed 1.4.4 in my virtual machine and it shows the above error. Afterwards, I removed 1.4.4 completely, installed 1.4.3, and everything works fine, without any error reported. Nothing else has been modified in the system otherwise. 
Maybe this can help narrow down the issue.

**Comment by amo13** [comment]
Alright, so I tried out even more to find out what is going on and I think I found the source of the issue:

If I do a fresh installation, call `migrate` and `createsuperuser` and start everything, then everything is fine.
Now, if I do the same, but after creating the superuser and before starting everything, I just comment out `#PAPERLESS_SECRET_KEY=change-me` in the configuration file by removing the `#`, then I get the above errors as soon as I drop a PDF onto the upload field of the webUI.

So what am I doing wrong here? 

I guess I just misunderstand the secret_key setting. What exactly is it meant to be good for (wasn't really able to understand that, even after googling =/ )?

**Comment by amo13** [comment]
Ok, so the error showed up after an update. The update pulls the `paperless.conf` template in the paperless directory. 

What I did not expect is that this template `paperless.conf` takes precedence before `/etc/paperless.conf`. So I got this bug because my paperless instance was using the template config instead of mine. 

So I must ask: shouldn't either the template `paperless.conf` file be renamed to `paperless.conf.template` or the precedence be changed to first `/etc/paperless.conf` and only second `./paperless.conf`?
